README.md

```diff
@@ -1,6 +1,6 @@
 # Chain.jl
 
-Even more convenient than pipes.
+A [Julia package](https://julialang.org/packages/) for chaining function calls using a more convenient syntax than Julia's native [piping functionality](https://docs.julialang.org/en/v1/manual/functions/#Function-composition-and-piping).
```

Comment:
it's not really `for chaining function calls`, more `for piping a value through a series of transformation expressions`

Reply:
Thanks for the correction :) I've applied this change.